test/components/groupTests.ts

```diff
@@ -1,258 +1,281 @@
-      assert.isFalse(cg.fixedWidth(), "width is not fixed if subcomponents are not fixed width");
-      assert.isFalse(cg.fixedHeight(), "height is not fixed if subcomponents are not fixed height");
+      assert.isFalse(group.fixedWidth(), "width is not fixed if subcomponents are not fixed width");
+      assert.isFalse(group.fixedHeight(), "height is not fixed if subcomponents are not fixed height");
 
-      cg.renderTo(svg);
-      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
-      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      group.renderTo(svg);
+      assert.strictEqual(group.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(group.height(), SVG_HEIGHT, "occupies all offered height");
       svg.remove();
     });
 
-    it("with fixed-size Components", () => {
-      let svg = TestMethods.generateSVG();
-      let tall = new Mocks.FixedSizeComponent(SVG_WIDTH / 4, SVG_WIDTH / 2);
-      let wide = new Mocks.FixedSizeComponent(SVG_WIDTH / 2, SVG_WIDTH / 4);
+    it("requests space correctly when it contains fixed-size Components", () => {
+      let tallComponent = new Mocks.FixedSizeComponent(SVG_WIDTH / 4, SVG_WIDTH / 2);
+      let wideComponent = new Mocks.FixedSizeComponent(SVG_WIDTH / 2, SVG_WIDTH / 4);
 
-      let cg = new Plottable.Components.Group([tall, wide]);
+      let group = new Plottable.Components.Group([tallComponent, wideComponent]);
 
-      let request = cg.requestedSpace(SVG_WIDTH, SVG_HEIGHT);
+      let request = group.requestedSpace(SVG_WIDTH, SVG_HEIGHT);
       assert.strictEqual(request.minWidth, SVG_WIDTH / 2, "requested enough space for widest Component");
       assert.strictEqual(request.minHeight, SVG_HEIGHT / 2, "requested enough space for tallest Component");
 
-      let constrainedRequest = cg.requestedSpace(SVG_WIDTH / 10, SVG_HEIGHT / 10);
+      let constrainedRequest = group.requestedSpace(SVG_WIDTH / 10, SVG_HEIGHT / 10);
       assert.strictEqual(constrainedRequest.minWidth, SVG_WIDTH / 2, "requested enough space for widest Component");
       assert.strictEqual(constrainedRequest.minHeight, SVG_HEIGHT / 2, "requested enough space for tallest Component");
 
-      cg.renderTo(svg);
-      assert.strictEqual(cg.width(), SVG_WIDTH, "occupies all offered width");
-      assert.strictEqual(cg.height(), SVG_HEIGHT, "occupies all offered height");
+      group.renderTo(svg);
+      assert.strictEqual(group.width(), SVG_WIDTH, "occupies all offered width");
+      assert.strictEqual(group.height(), SVG_HEIGHT, "occupies all offered height");
+      svg.remove();
+    });
+
+    it("allocates space to its Components correctly", () => {
+      let FIXED_COMPONENT_SIZE = SVG_WIDTH / 4;
+      let fixedComponent = new Mocks.FixedSizeComponent(FIXED_COMPONENT_SIZE, FIXED_COMPONENT_SIZE)
+                                    .xAlignment("right").yAlignment("bottom");
```

Comment:
I don't like this type of indentation because it can easily get off sync when global find / replace method names